testutil/src/main/java/org/spine3/test/NullToleranceTest.java

```diff
@@ -0,0 +1,393 @@
+         * the information about the {@code Preconditions} method.
+         *
+         * <p> The second {@code StackTraceElement} should contains information
+         * about the method from the {@code targetClass}.
+         *
+         * @param cause the {@code Throwable}
+         * @return {@code true} if the {@code StackTraceElement}s matches the expected, {@code false} otherwise
+         */
+        private boolean isExpectedStackTraceElements(Throwable cause) {
+            final StackTraceElement[] stackTraceElements = cause.getStackTrace();
+            final StackTraceElement preconditionsElement = stackTraceElements[0];
+            final boolean preconditionClass = Preconditions.class.getName()
+                                                                 .equals(preconditionsElement.getClassName());
+            if (!preconditionClass) {
+                return false;
+            }
+
+            final StackTraceElement expectedUtilClassElement = stackTraceElements[1];
+            final boolean correct = isCorrectMethodName(method.getName(), expectedUtilClassElement.getMethodName());
+            if (!correct) {
+                return false;
+            }
+
+            final boolean correctClass = targetClassName.equals(expectedUtilClassElement.getClassName());
+            return correctClass;
+        }
+
+        private static boolean isCorrectMethodName(String expectedMethodName, String actualMethodName) {
+            final boolean correct = expectedMethodName.equals(actualMethodName);
+            return correct;
+        }
+    }
+
+    /**
+     * A builder for producing the {@link NullToleranceTest} instance.
+     */
+    public static class Builder {
+
+        private Class targetClass;
+        private final Set<String> excludedMethods;
+        private final Map<? super Class, ? super Object> defaultValues;
+        private final Pattern pattern;
+        private static final String REGEX = "^[a-zA-Z]+$";
+
+        private Builder() {
+            defaultValues = newHashMap();
+            excludedMethods = newHashSet();
+            pattern = Pattern.compile(REGEX);
+        }
+
+        /**
+         * Sets the utility class.
+         *
+         * @param utilClass the utility {@link Class}
+         * @return the {@code Builder}
+         */
+        public Builder setClass(Class utilClass) {
+            this.targetClass = checkNotNull(utilClass);
+            return this;
+        }
+
+        /**
+         * Adds the method name which will be excluded during the check.
+         *
+         * @param methodName the name of the excluded method
+         * @return the {@code Builder}
+         */
+        @SuppressWarnings("WeakerAccess") // Will be used outside the package
+        public Builder excludeMethod(String methodName) {
+            checkNotNull(methodName);
+            final Matcher matcher = pattern.matcher(methodName);
```

Comment:
Have you checked [this piece of documentation](http://docs.oracle.com/javase/8/docs/api/javax/lang/model/SourceVersion.html#isName-java.lang.CharSequence-) I posted previously?